### PR TITLE
{tools}[GCCcore/10.3.0] git v2.32.0

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2617.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/b080d8eb31c221d21b11aae1f6e85553 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
select-pika-c6gd-2xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/b5d7f21693e852f3144cdc3637c552bf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild2.novalocal - Linux centos linux 8.3.2011, POWER, IBM pSeries (emulated by qemu) (power9le), Python 3.6.8
See https://gist.github.com/a424832c24c77b7db6e923bfb6c59979 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/b2e1784ffed99691a50a111e715a6e21 for a full test report.

File: easybuild/easyconfigs/g/git/git-2.32.0-GCCcore-10.3.0-nodocs.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+]
+
+dependencies = [
+    ('cURL', '7.76.0'),
+    ('expat', '2.2.9'),
+    ('gettext', '0.21'),
+    ('Perl', '5.32.1'),
+]
+
+osdependencies = [OS_PKG_OPENSSL_DEV]
```
Comment (boegel) [inline easybuild/easyconfigs/g/git/git-2.32.0-GCCcore-10.3.0-nodocs.eb:33]: @robert-mijakovic Please use the new `OpenSSL` wrapper instead, so:

* add `('OpenSSL', '1.1', '', True)` to `dependencies`
* drop the `osdependencies`

I'll look into enhancing the easyconfig test suite to catch this...

(I also overlooked this in my own PR...)

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/g/git/git-2.32.0-GCCcore-10.3.0-nodocs.eb (hunk 1 above)
Comment (robert-mijakovic) [reply easybuild/easyconfigs/g/git/git-2.32.0-GCCcore-10.3.0-nodocs.eb:33]: Will fix it. I have noticed in the morning that I had such a mistake in one of the other ECs I have in my stack to commit.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/5c2050f0185fe4966d5e3aa30f6c8c77 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @robert-mijakovic!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
select-pika-c6gd-2xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/8398d09cc8349a922716ba3a1182e867 for a full test report.

**Pull request merged**